go/vt/vtgate/executor.go

```diff
@@ -1136,6 +1137,7 @@ func (e *Executor) StreamExecute(ctx context.Context, method string, safeSession
 	}
 	query, comments := sqlparser.SplitMarginComments(sql)
 	vcursor, _ := newVCursorImpl(ctx, safeSession, comments, e, logStats, e.vm, e.VSchema(), e.resolver.resolver)
+	vcursor.SetIgnoreMaxMemoryRows(true)
```

Comment:
Calling this out since this may be easy to miss -- setting this explicitly to true since this is being called within `StreamExecute`